### PR TITLE
2023

### PR DESCRIPTION
**Pull request by michael-grace**

### CODE REVIEW

**Review by markspolakovs — APPROVED** [review]
ok

**Comment by ColinRoitt** [comment]
lgtm

File: index.html [hunk 1]
```diff
@@ -41,13 +41,13 @@ <h1>no</h1>
         const isToday = () => {
             // 2nd March 2021
             console.log('The Letter L');
```
Comment (markspolakovs) [inline index.html:43]: what

Comment (michael-grace) [reply index.html:43]: L

Comment (ColinRoitt) [reply index.html:43]: dont look at that 
its a secret

**Pull request merged**